crash-android-lib/src/main/kotlin/co/electriccoin/zcash/crash/android/internal/local/LocalCrashReporter.kt

```diff
@@ -0,0 +1,48 @@
+package co.electriccoin.zcash.crash.android.internal.local
+
+import android.content.Context
+import androidx.annotation.AnyThread
+import co.electriccoin.zcash.crash.ReportableException
+import co.electriccoin.zcash.crash.android.internal.CrashReporter
+import co.electriccoin.zcash.spackle.LazyWithArgument
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
+
+/**
+ * Registers an exception handler to write exceptions to disk.
+ */
+internal class LocalCrashReporter(private val applicationContext: Context) : CrashReporter {
+
+    private val crashReportingScope = CoroutineScope(SupervisorJob() + Dispatchers.Main)
```

Comment:
shouldn't we cancel this scope at some point?

Reply:
At what point would you suggest?  Crash reporting as currently implemented is always registered and continues for the lifetime of the application.

Reply:
i think we should do it when the app is terminated